routes/authRouter.js

```diff
@@ -0,0 +1,16 @@
+const Router = require('express')
+const router = new Router()
+const authController = require('../controllers/authController')
+const {check} = require('express-validator')
+const authMiddleware = require('../middleware/authMiddleware')
+
+router.post('/registration', [
+	check('email', 'Email field cannot be empty').notEmpty(),
+	check('firstName', 'firstName field cannot be empty').notEmpty(),
```

Comment:
For the 'user-friendly' response it's better to use plain definition instead of mentioning 'technical' things
```suggestion
	check('firstName', 'First name cannot be empty').notEmpty(),
```